.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - vendor/**/*
+    - gemfiles/**/*
+
+Metrics/LineLength:
+  Max: 140
+
+# Can't do without a little table-flip there ;)
+Style/AsciiComments:
+  Exclude:
+    - 'test/scope_test.rb'
```

Comment:
I actually think it's a bad rule for this project. It excludes a bevy of common words with accents that are used by english speakers. On more massive projects with more regular contributions I think it makes sense.